File: Java/BookStoreOOP/src/model/Bill.java
package model;

import java.util.Date;

public class Bill {
    private String id;
    private Date date;
    private String status;
    private Reader reader;
    private Staff staff;

    public Bill(String id, Date date, String status, Reader reader, Staff staff) {
        this.id = id;
        this.date = date;
        this.status = status;
        this.reader = reader;
        this.staff = staff;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Reader getReader() {
        return reader;
    }

    public void setReader(Reader reader) {
        this.reader = reader;
    }

    public Staff getStaff() {
        return staff;
    }

    public void setStaff(Staff staff) {
        this.staff = staff;
    }
}

File: C#/BookStoreOOP/BookStoreOOP/Model/Bill.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookStoreOOP.Model
{
    public class Bill
    {
        private string id;
        private DateTime date;
        private string status;
        private Reader reader;
        private Staff staff;

        public Bill(string id, DateTime date, string status, Reader reader, Staff staff)
        {
            this.id = id;
            this.date = date;
            this.status = status;
            this.reader = reader;
            this.staff = staff;
        }

        public string getId()
        {
            return id;
        }

        public void setId(string id)
        {
            this.id = id;
        }

        public DateTime getDate()
        {
            return date;
        }

        public void setDate(DateTime date)
        {
            this.date = date;
        }

        public string getStatus()
        {
            return status;
        }

        public void setStatus(string status)
        {
            this.status = status;
        }

        public Reader getReader()
        {
            return reader;
        }

        public void setReader(Reader reader)
        {
            this.reader = reader;
        }

        public Staff getStaff()
        {
            return staff;
        }

        public void setStaff(Staff staff)
        {
            this.staff = staff;
        }
    }
}
